c10/util/complex_type.h

```diff
@@ -561,6 +561,10 @@ C10_HOST_DEVICE c10::complex<T> polar(const T& r, const T& theta = T()) {
 } // namespace c10
 
 // math functions are included in a separate file
+#define C10_INTERNAL_INCLUDE_COMPLEX_MATH_H
```

Comment:
Would it be a bit cleaner if we define a single macro `C10_INTERNAL_INCLUDE_COMPLEX_REMAINING_H` and use it both in `c10/util/complex_utils.h` and `c10/util/complex_math.h`